src/modules/admin/menu/console/AddOnModal.tsx

```diff
@@ -72,12 +71,12 @@ const AddOnModal: React.FC<AddOnModalProps> = ({
         <ModalHeader>
           <H4>Add-on</H4>
           <ModalActionContainer>
```

Comment:
I don't think it's necessary for me. But in case you guys want the term `Add-on` to be translated, I'll just translate it as `ส่วนขยาย` then.

Please confirm if we're gonna translate it or not, so I can continue to work on it.

Reply:
ส่วนขยาย is so sodigvnsaopbignhswprob 
so ท็อปปิ้ง ??

Reply:
or just ignored it coz it in admin menu. admin must have some knowledge right?  🥵

Reply:
@lazykern

Reply:
> admin must have some knowledge

Nobody knows
...
I'll just call it `ท็อปปิ้ง` just like you mentioned then